packages/twenty-server/src/engine/core-modules/guard-manager/services/guard-error-manager.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@nestjs/common';
+
+import { DomainManagerService } from 'src/engine/core-modules/domain-manager/services/domain-manager.service';
+import { getCallerByContext } from 'src/engine/core-modules/guard-manager/utils/get-caller-by-context';
+
+@Injectable()
+// eslint-disable-next-line @nx/workspace-inject-workspace-repository
+export class GuardErrorManagerService {
+  constructor(private readonly domainManagerService: DomainManagerService) {}
+
+  dispatchErrorFromGuard(
```

Comment:
I don't understand internal vs external. This looks a bit overkill to me, why not just redirecting?